GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, lead, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to
+be the people who contribute, review, guide, and collectively make decisions
+about the direction and scope of the project.
+
+A typical maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing code &mdash; a lot of code, especially "glue code" or "grunt work";
+fixing bugs, improving processes, making judgments on dependencies, handling
+security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve

```

Comment:
I worry that this one doesn't quite fit with the others. Voting on conflicts and decisionmaking is a privilege and a duty of a maintainer, not just a thing that someone typically does before being considered for maintainership, right? It might make more sense to call out the duty of the maintainer in a separate list than the "typical behavior of the maintainer" list here.